Standards/scs-0106-v1-evaluate-current-state-of-cloudmon.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Evaluating the deployment of CloudMon in SCS infrastructure
+type: Decision Record
+---
+
+## Introduction
+
+In the fast-paced environment of modern cloud computing, ensuring the reliability and performance
+of cloud infrastructure is paramount for organizations. Effective monitoring and testing framework
+are essential tools in this endeavor, enabling proactive identification and resolution of issues before
+they impact critical operations. However, selecting the right monitoring solution can be a daunting task,
+particularly when faced with options that are overly complex and lack essential documentation and working
+scenarios.
+
+## Motivation
+
+One such solution, the [Cloudmon](https://stackmon.org/) project, presents challenges that may hinder its
+suitability for organizations seeking efficient and reliable monitoring capabilities. This introduction
+outlines the reasons why our organization opts against utilizing the Cloudmon project and instead embraces
```

Comment:
this sentence sounds more like a decision and not like a motivation

Reply:
let me rephrase that, thanks